products/slackware.md

```diff
@@ -37,7 +36,7 @@ releases:
     releaseDate: 2013-11-04
 -   releaseCycle: "14.0"
     eol: false
-    releaseDate: 20l2-09-26
+    releaseDate: 2012-09-26
```

Comment:
The validation didn't run on your PR due to GitHub restrictions, but it caught this on my PR. Did you copy this from somewhere by any chance? Would be nice to correct it upstream as well.